README.md

```diff
@@ -27,7 +27,7 @@ The BIDS Validator can be run with the [Deno][] runtime
 (see [Deno - Installation][] for detailed installation instructions):
 
 ```shell
-deno run -ERN jsr:@bids/validator
+deno run -ERNW jsr:@bids/validator
```

Comment:
Just because RW is a meaningful sequence:

```suggestion
deno run -ERWN jsr:@bids/validator
```

And note that the meanings of `-E`, `-R` and `-N` are described in the next paragraph. Could you add `-W`?

Reply:
Also we could mention that `-A` allows all permissions, if this is getting long.

Reply:
done for 1st comment. I would not bother with `-A` as not used here/not directly relevant to this PR ;)